www/generate_tutorial/src/input/tutorial.md

```diff
@@ -398,6 +398,50 @@ The `fromScratch` and `fromOriginal` records are equal, although they're defined
 Note that `&` can't introduce new fields to a record, or change the types of existing fields.
 (Trying to do either of these will result in an error at build time!)
 
+## [Optional Record Fields](#optional-record-fields) {#optional-record-fields}
+
+Roc supports optional record fields using the `?` operator. This can be a useful pattern where you pass a function a record of configuration values, some of which you don't really care about and want to leave as defaults.
+
+In Roc you can write a function like;
+
+<pre><samp>table <span class="kw">=</span> <span class="kw">\</span><span class="paren">{</span> height, width, title<span class="colon">?</span> "", description<span class="colon">?</span> "" <span class="paren">}</span> <span class="kw">-></span>
```

Comment:
maybe worth filling in the strings with real values, like "Table" and "A wooden table"? Just to show this is more useful than using an empty string in the way you would use `Optional := [Some Str, None]`.